testing/python_lib/integration_base.py

```diff
@@ -118,8 +126,12 @@ def _write_faucet_config(self, config):
         return self._write_yaml_to_file(filename, config)
 
     def _get_faucet_config_path(self):
-        return os.path.dirname(os.path.abspath(__file__)) + \
-            '/../../inst/forch-faucet-1/faucet/faucet.yaml'
+        if self.stack_options.get('fot'):
+            return os.path.dirname(os.path.abspath(__file__)) + \
+                '/../../inst/forch-controller-1/faucet/faucet.yaml'
```

Comment:
replace common string with format placeholder constant (../inst/%s/) and then use % substitution

Reply:
Addressed.